glean-core/ios/Glean/Glean.swift

```diff
@@ -119,6 +120,10 @@ public class Glean {
                 _ = self.setDebugViewTag(self.debugViewTag!)
             }
 
+            if self.logPings != nil {
+                _ = self.setLogPings(self.logPings!)
```

Comment:
Since `setLogPings` doesn't return anything, you should be able to get rid of the `_ =` part.  This pattern is just used to satisfy the compiler that we _know_ that the function is returning a value and we are explicitly ignoring it.

Reply:
Another more swift-like way of writing this might be:
```Swift
if let logPings = self.logPings {
    self.setLogPings(logPings)
}
```

This is a form of optional unwrapping that only falls into the condition if the value is not nil.  Assignment of an optional value in swift returns true if there is a value or false if it is nil.

Reply:
>  This pattern is just used to satisfy the compiler that we know that the function is returning a value and we are explicitly ignoring it.

Oops, copy pasted from the `debugViewTag` part and didn't realize this was not necessary. Good catch.